serverless/monitoring-serverless.adoc

```diff
@@ -0,0 +1,11 @@
+[id="monitoring-serverless"]
+= Monitoring Serverless applications
+include::modules/common-attributes.adoc[]
+:context: cluster-monitoring-serverless
+
+As an {product-title} cluster administrator, you can deploy the {product-title} monitoring stack and monitor the status of each deployment as well as metrics data.
+
+When using the OpenShift Serverless operator, the required ServiceMonitor objects are created automatically for cluster monitoring on Serverless applications. OpenShift Serverless components, such as Knative Serving, expose metrics data, and administrators can monitor this data by using the {product-title} Web console.
```

Comment:
"web console" should be lowercase.

Reply:
Look out for this in other docs because I used it because I saw it elsewhere and thought it was a thing :joy: if I find where it's in other docs again I'll let you know.